### PR TITLE
compiles cc clvm when updated

### PR DESCRIPTION
**Pull request by wbendick**

### CODE REVIEW

File: src/wallet/cc_wallet/cc_wallet_puzzles.py [hunk 1]
```diff
@@ -30,9 +32,12 @@ def cc_make_core(originID):
...
-
+    compile_clvm('src/wallet/cc_wallet/coloured_coins.clvm', 'src/wallet/cc_wallet/coloured_coins.clvm.hex')
```
[the remark this answers is not in the document]
Comment (wbendick) [reply src/wallet/cc_wallet/cc_wallet_puzzles.py:35]: yes, compile_clvm already makes the check for us

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+-e git+https://git@github.com/Chia-Network/clvm_tools.git@2d0e00a87c471166fd0886609ee141940d6a7e27#egg=clvm_tools
```
[the remark this answers is not in the document]
Comment (wbendick) [reply requirements-dev.txt]: removed

**Comment by mariano54** [comment]
Update @wbendick ?

**Comment by mariano54** [comment]
Closing due to inactivity

**Pull request closed without merging**